src/PrefsManager.h

```diff
@@ -209,6 +209,7 @@ class PrefsManager
 	Preference<AllowW1>	m_AllowW1; // this should almost always be on, given use cases. -aj
 	Preference<bool>	m_bEventMode;
 	Preference<int>	m_iCoinsPerCredit;
+	Preference<int>	m_iMaxNumCredits; // Should be able to change max number of credits, why cap? - CrashCringle
```

Comment:
I think the comment isn't needed so would prefer to remove.

Reply:
Ah, that was a leftover comment. Honestly forgot it was there.